src/marker.rs

```diff
@@ -309,7 +309,7 @@ pub use nightly::Ungil;
 /// - It provides a global API for the Python interpreter, such as [`Python::eval_bound`].
 /// - It can be passed to functions that require a proof of holding the GIL, such as
 ///   [`Py::clone_ref`].
-/// - Its lifetime represents the scope of holding the GIL which can be used to create Rust
+/// - (Deprecated) Its lifetime represents the scope of holding the GIL which can be used to create Rust
```

Comment:
I feel like this change should probably be a separate PR. I also think rather than marking as deprecated we should update the wording to mention other GIL bound types (like `Bound`)

Reply:
Ooops, this was lying around in my working dir and didn't notice it.